### DISCUSSION TITLE
Is it possible to divide the .tif directory into smaller to be run conversion NAD83 to WGS84

### DISCUSSION THREAD

**Discussion by sambk** [open-afc-project/openafc#239] [Q&A]
Hi openafc team,
My disk is not enough 600GB to run conversion NAD83 to WGS84 (https://github.com/open-afc-project/openafc/blob/main/database_readme.md#detailed-instructions-for-downloading-and-converting-3dep-files-for-use-in-afc) so I have an idea to split the folder into smaller ones. But not all files are converted, the number of output files are therefore less than the original ones. I'm not sure whether it is because I split the directory. Could anyone please help?

When converting to 1_arcsec_wgs84_datum_aligned:
> ERROR 1: /mnt/nfs/rat_transfer/3dep/1_arcsec_nad83/USGS_1_n39w114.tif:Using code not yet in table
ERROR 1: TIFFReadEncodedTile() failed.
ERROR 1: /mnt/nfs/rat_transfer/3dep/1_arcsec_nad83/USGS_1_n39w114.tif, band 1: IReadBlock failed at X offset 4, Y offset 2: TIFFReadEncodedTile() failed.

When converting to 1_arcsec_wgs84:
> to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n07e152.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w107.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w089.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w115.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w108.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w093.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n16w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w102.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w104.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n16w096.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w111.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w090.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n08e158.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w100.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n16w097.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w092.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w104.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w104.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w094.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w089.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w093.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w102.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w100.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w104.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w099.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w090.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w103.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w090.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w102.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w111.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w106.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w100.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w097.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w105.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w093.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w091.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w106.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w106.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n08e152.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w095.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w097.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w101.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w091.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w099.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w103.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w099.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w105.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w101.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w089.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w101.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w102.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w104.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w110.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w092.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w088.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n07e134.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w100.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w096.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w100.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w099.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w099.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w094.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w105.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w090.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n10e138.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w097.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w105.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w094.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w105.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w097.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w101.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w110.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n06e162.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w091.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n08e151.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w102.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w092.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w096.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w107.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w091.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w091.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w099.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w101.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w111.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n07e158.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w092.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w099.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w101.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w096.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w095.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w100.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w089.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n16w094.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w103.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w103.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n08e134.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w087.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n06e163.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n16w093.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w106.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w106.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w100.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n09e134.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w097.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w088.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w102.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w096.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w089.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w100.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w098.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w088.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n20w103.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w102.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n07e151.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n24w107.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w090.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n15w093.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w101.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w105.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n23w103.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n16w092.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n22w104.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w103.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w099.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w101.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n17w095.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w087.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n18w091.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n19w112.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n21w088.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n84w077.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n84w070.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n84w076.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n84w078.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n84w075.tif' not covered by any source geoid. It will not be processed
to_wgs84.py. WARNING: '/mnt/nfs/rat_transfer/3dep/1_arcsec_wgs84_datum_aligned/USGS_1_n84w069.tif' not covered by any source geoid. It will not be processed

**Comment by farinazedalat** [comment]
Hello,
Unfortunately, RKF won't be able to review and respond to such issues anymore unless someone else can help out. If you'd like to contract RKF to help resolve all your issues with bringing up your open AFC instance, we can certainly do that.

**Comment by sambk** [comment]
Thank you @farinazedalat! I will consider it

**Comment by sambk** [comment]
Hi @farinazedalat,
Could you please tell me how to purchase a support from RKF?

Thank you

**Comment by farinazedalat** [comment]
This questions was resovled outside OpenAFC.
